src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.1.27
+++++++
+* appservice: list-location: reports Free as an invalid SKU
```

Comment:
I thought this change actually fixes the issue where it reports Free as invalid.
This note makes it seem like now it will report Free as invalid.